concordium-contracts-common/src/schema_json.rs

```diff
@@ -1048,10 +1269,10 @@ impl Type {
             Self::I64 => "<Int64>".into(),
             Self::I128 => "<Int128>".into(),
             Self::ILeb128(size) => {
-                format!("String of size at most {size} containing a signed integer.").into()
+                format!("<String of size at most {size} containing a signed integer.>").into()
```

Comment:
The size here is a bit more complex and a bit hard to explain, but it is the max number of bytes for the leb128 encoding, the user types in the integer (wrapped in a string) and not the encoding, so not sure we can communicate the size in a good way